src/main/app/src/app/zaken/zaak-view/zaak-view.component.ts

```diff
@@ -148,6 +148,8 @@ export class ZaakViewComponent
   private zaakBesluitenListener!: WebsocketListener;
   private zaakTakenListener!: WebsocketListener;
   private datumPipe = new DatumPipe("nl");
+  private wasEinddatumGeplandExceededAtClose: boolean = false;
+  private wasUiterlijkeEinddatumAfdoeningExceededAtClose: boolean = false;
```

Comment:
Do we need these variables? I do not see them being used?

Either way, we do not need to type, this is inferred from the default type

```suggestion
  private wasEinddatumGeplandExceededAtClose = false;
  private wasUiterlijkeEinddatumAfdoeningExceededAtClose = false;
```